backends-velox/src/test/scala/org/apache/gluten/execution/TestOperator.scala

```diff
@@ -351,14 +351,20 @@ class TestOperator extends VeloxWholeStageTransformerSuite with AdaptiveSparkPla
   }
 
   test("window expression") {
-    Seq("sort", "streaming").foreach {
-      windowType =>
+    Seq(("sort", 0), ("streaming", 1)).foreach {
```

Comment:
@zml1206 Why the local sort size is different in "sort" and "streaming" ?

Reply:
Velox StreamingWindow need to require child order, sort not need.

Reply:
https://github.com/apache/incubator-gluten/blob/94b79c7a8a3166d764c8d405a6aa3225365e924f/gluten-core/src/main/scala/org/apache/gluten/execution/WindowExecTransformer.scala#L69